pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -177,6 +178,8 @@ rec {
           userHook = config.stdenv.userHook or null;
           __ignoreNulls = true;
 
+          inherit strictDeps;
```

Comment:
This bloats every derivation with a yet another extra field and makes `mkDerivation` slower. This is explicitly discouraged.